NitroxClient/MonoBehaviours/NitroxBootstrapper.cs

```diff
@@ -12,6 +12,11 @@ public void Awake()
 
             gameObject.AddComponent<MainMenuMods>();
             gameObject.AddComponent<CodePatchManager>();
+
+#if DEBUG
+            gameObject.AddComponent<ConsoleJoinServer>();
```

Comment:
The console resets registered commands when the game loads; as such the commands registered in `ConsoleJoinServer` are only available in the menu (where they shouldn't be), not in the game itself (where they should be).

Reply:
Interesting. I assumed that this persisted into the game itself. No matter - just need to create a new game object and set it to not be destroyed on load. It will simply be alive and well for the life of the client - but will stop/start multiplayer when it is appropriate to do so.

Objections?